Python_DataEngineering/Extract_Store_Fetch.py

```diff
@@ -0,0 +1,158 @@
+import requests
+from datetime import datetime
+from dateutil import tz
+import mysql.connector
+import schedule
+import time
+
+#dataBaseConfig function configures the DataBase connectivity, returns the cursor pointing to the database 
+def databaseConfig():
+    database = "bikeapi"
+    username = 'root'
+    password = 'XXXkalia@123'
+
+    # Open database connection
+    db = mysql.connector.connect(host='localhost', user=username, passwd=password, db=database)
+
+    # prepare a cursor object using cursor() method
+    cursor = db.cursor()
+    
+    return (cursor,db)
+
+#This function converts UTC Time provided in API to IST using Python package dateutil and return the IST date and time as a String
+def convertTOIST(datetime_in_UTCs):

```

Comment:
Nice thought process here!

Reply:
Ideally, this should not be done before putting things inside the DB, as DB timestamps should be TZ agnostic. When the data is being fetched from DB to give to user, we can run this based on the users TZ.

Reply:
Yeah i thought the same initially. But the obly thing i considered is IST time as i assumed that users will be from India. But that is wrong. I got what you have said. I will do the requested changes here .